sormas-ui/src/main/java/de/symeda/sormas/ui/campaign/campaignstatistics/CampaignStatisticsView.java

```diff
@@ -185,7 +188,7 @@ public CampaignStatisticsFilterForm createFilterForm() {
 				for (CampaignFormElement element : campaignFormElements) {
 					if (element.isImportant() || !onlyImportantFormElements) {
 						String type = element.getType();
-						if (type.equals("number") || type.equals("yes-no")) {
+						if (type.equals(ELEMENT_TYPE_NUMBER) || type.equals(ELEMENT_TYPE_YES_NO)) {
```

Comment:
Please use `CampaignFormElementType.fromString` for the comparison because it's case-insensitive, so:

```
if (CampaignFormElementType.fromString(type) == CampaignFormElementType.NUMBER ...
```